README.md

```diff
@@ -4,8 +4,26 @@
 ![CI-Status](https://travis-ci.com/livefront/sealed-enum.svg?token=WuckG7RmGXoNTzG4QzVk&branch=master)
 [![Release](https://jitpack.io/v/livefront/sealed-enum.svg)](https://jitpack.io/#livefront/sealed-enum)
 
+Generates enum-like behavior for sealed classes of objects:
```

Comment:
👍 Cool

I suggest one more tiny thing, 

```
Generates enum-like behavior for sealed classes of objects.
Just annotate the `companion` with `@GenSealedEnum`, and you'll see generated extensions like `ordinal` and `values`:
```

Reply:
Just force pushed with that second line!